internal/mesh/internal/controllers/gatewayproxy/controller.go

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) HashiCorp, Inc.
+//
```

Comment:
Is this supposed to be here? I figured the header robot thing is pretty particular about it

Reply:
not 100% on that, lemme re-run the copywrite stuff again